packages/main/src/themes/Label.css

```diff
@@ -77,3 +77,9 @@ bdi {
 	margin-right: .125rem;
 	margin-left: 0;
 }
+
+.ui5-label-root:focus {
+	outline-offset: -1px;
+	outline: 1px dotted var(--sapContent_FocusColor);
+	text-decoration: underline;
```

Comment:
should be set to ui5-label-text-wrapper otherwise it doesn't affect the text. Also I think we should skip this or to add shadow part to the the text wrapper and to style it from styles of breadcrumbs since it is ui5-breadcrumbs specific

Reply:
the rule for "text-decoration" was added by me by mistake, I removed it as it is not needed at all

so I just tried to move this css to Breadcrumbs.css, but I do not seem to be allowed to target ui5-label-root from there

I need to target ui5-label-root as this is the element that has the tabIndex attribute (the inner ui5-label-text-wrapper is not focusable, sorry for overlooking the confusing "text-decoration" rule)